samples/Ch11_vectors/fig_11_14_marray.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2023 Intel Corporation
+
+// SPDX-License-Identifier: MIT
+
+#include <cassert>
+#include <sycl/sycl.hpp>
+
+using namespace sycl;
+
+template <typename T, size_t N>
+bool checkEqual(marray<T, N> A, marray<T, N> B) {
+  for (int i = 0; i < N; i++) {
+    if (A[i] != B[i]) {
+      return false;
+    }
+  }
+  return true;
+}
+
+int main() {
+  queue q;
+  marray<float, 4> input{1.0004f, 1e-4f, 1.4f, 14.0f};
+  marray<float, 4> res[2] = {{-1, -1, -1, -1}, {-2, -2, -2, -2}};
+  {
+    buffer<marray<float, 4>, 1> in_buf(&input, 1);
+    buffer<marray<float, 4>, 1> re_buf(&res[0], sycl::range<1>(2));
```

Comment:
Can these be made any more concise?  We can lose `sycl::` at the very least.

Reply:
Something like this?

```suggestion
    buffer in_buf(&input, range{1});
    buffer re_buf(res, range{2});
```